src/stories/Library/list-buttons/list-buttons.scss

```diff
@@ -29,4 +30,11 @@
       display: flex;
     }
   }
+  .hide-on-mobile {
+    display: none;
+
+    @include breakpoint-s {
+      display: flex;
+    }
+  }
```

Comment:
I am a BEM novice but if you wanted a buttons modifier for hiding on mobile wouldn't it be something like this?:

```
 &__buttons--hide-on-mobile {
    display: none;
    @include breakpoint-s {
      display: flex;
    }
  }